applications/realtime-heatmap/helm/templates/heatmap-deploy.yaml

```diff
@@ -84,6 +84,8 @@ spec:
           - |
             python /ponchik2/src/generate_streams.py $STREAM_COUNT /ponchik2/templates/data.json && /ponchik2/edge/start.sh
           env:
+          - name: HEATMAP_ID
```

Comment:
Please provide the JIRA details to understand this change in the commit message.

Reply:
Updated